plenum/test/conftest.py

```diff
@@ -58,6 +58,12 @@
 from plenum.test.test_node import TestNode, TestNodeSet, Pool, \
     checkNodesConnected, ensureElectionsDone, genNodeReg
 from plenum.common.config_helper import PConfigHelper, PNodeConfigHelper
+import json
```

Comment:
please, move that to the beginning of the file

Reply:
done